pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -21,15 +21,14 @@ import (
 	"math"
 
 	"k8s.io/client-go/util/workqueue"
+	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
-const parallelism = 16
-
 // chunkSizeFor returns a chunk size for the given number of items to use for
 // parallel work. The size aims to produce good CPU utilization.
-func chunkSizeFor(n int) workqueue.Options {
-	s := int(math.Sqrt(float64(n)))
-	if r := n/parallelism + 1; s > r {
+func chunkSizeFor(n int, in *KubeSchedulerConfiguration) workqueue.Options {
```

Comment:
I guess you mean `config.KubeSchedulerConfiguration`?

Reply:
I did, the problem is that the `config.KubeSchedulerConfiguration` is a struct, and I am unable to get the currently loaded configuration of the scheduler via a function call. I see that its propagated via function variables